docs/website/mtouch-errors.md

```diff
@@ -639,6 +639,12 @@ This warning won't affect the application being built, however you might not be
 
 Please report this issue to the publisher of the assembly package (e.g. nuget author) so this can be fixed in their future releases.
 
+### <a name="MT0132/>MT0132: Unknown optimization: *. Valid values are: *
+
+The specified optimization was not recognized.
+
+The accepted format is `[+|-]optimization-name`, where `optimization-name` is one of the values listed in the error message.
+
```

Comment:
We'll need more documentation that this - and the error should point to that (new) document.
This is likely a document we should have inside our repo (like this one) and update the main documentation repo on release. We just need to define where it should be located/named to avoid conflicts later.

@conceptdev comment ? also I'm not sure what's Brad GitHub handle

Reply:
@spouliot documented: 81109649c147c70ffb4766595a751acf44868fe2 (still missing the final location of the document though, so I can't update the error help).

Reply:
Hey @rolfbjarne, looks like this should go in **Guides > Cross-Platform > iOS & Mac** since it's relevant to both Mac and iOS. For a name, how about: _Build Optimizations_?

Reply:
I'll create a different PR for the documentation changes, this PR is already long, I need these changes for other pending working, and the documentation can be updated independently.